### ISSUE TITLE
Stötter koden golang 1.18?

### ISSUE THREAD

**Issue by joberget** [nrkno/terraform-registry#39]
https://github.com/nrkno/terraform-registry/blob/9e39aa11d7821dfbf321e25da968d915b15ee03a/Dockerfile#L5

**Comment by stigok** [comment]
Yep! I have a branch for upgrading to 1.18 already. It will pop up here soon :)

PS. Please keep the language to English so our future consumers also will understand :)
